src/contrib/cluster/Akka.Cluster.Tools/Client/reference.conf

```diff
@@ -94,6 +94,19 @@ akka.cluster.client {
   # Turning this setting to off or false will cause the ClusterClient messages
   # to be serialized using the ClusterClientMessageSerializer and not the default Object serializer. 
   use-legacy-serialization = on
+  
+  use-initial-contacts-discovery = false
+  
+  discovery
+  {
+    method = <method>
```

Comment:
If method is null, empty, whitespace, or "\<method>" (not set), we'll use the default akka.discovery.method value. If that value also not set, we fall back to "config" method.